### ISSUE TITLE
WebGL trace UI not rendering correctly

### ISSUE THREAD

**Issue by wesalvaro** [codegl/tracegl#48]
See included screenshot for issue.

Host is a VM Ware Virtual Machine of Ubuntu Precise Pangolin.
Client:
- Browsers Tested: Chrome Stable (up-to-date), Safari 6.0.4
- Model Name:   Mac Pro
- Model Identifier: MacPro5,1
- Processor Name:   Quad-Core Intel Xeon
- Processor Speed:  3.2 GHz
- Number of Processors: 1
- Total Number of Cores:    4
- L2 Cache (per Core):  256 KB
- L3 Cache: 8 MB
- Memory:   16 GB
- Processor Interconnect Speed: 4.8 GT/s
- Boot ROM Version: MP51.007F.B03
- SMC Version (system): 1.39f11
- SMC Version (processor tray): 1.39f11

![screen shot 2013-05-01 at 4 00 49 pm](https://f.cloud.github.com/assets/13286/451610/14e87ff2-b2b3-11e2-8a96-e7f71ce8f2d3.png)

**Comment by wesalvaro** [comment]
Video Card: ATI Radeon HD 5770 1024 MB graphics

**Comment by sethyuan** [comment]
I have this problem too, mine is a 15-inch, Early 2011 MacBook Pro with AMD Radeon HD 6750M 1024 MB graphic card.

**Comment by codegl** [comment]
Yeah this is the ATI problem, i'm really trying to recreate it / find someone with an ATI. I have now found some iMacs i can go try, and someone also reported it on vmware fusion / windows7 on retina macbook with nvidia which i'm trying to duplicate.

**Comment by DinisCruz** [comment]
I just had this exact same prob on chrome running inside vmware fusion / windows7 on retina macbook

**Comment by codegl** [comment]
I have traced this bug to be 'all varyings are corrupted in POINT sprites' on certain videocards / virtualisation envs (vmware fusion) There is no solution but for me to re-implement the text engine, which is what i'm doing.

**Comment by tanepiper** [comment]
Oh wow, having searched around it seems ATi hardware has always sucked at this, especially on Mac drivers/hardware.

Event RMS hates them:

![RMS Hates ATi](https://f.cloud.github.com/assets/376930/502509/b629037a-bcb1-11e2-9fb5-608c08138c79.jpg)

Shame :(
